tools/buildsys/src/manifest.rs

```diff
@@ -344,42 +355,93 @@ pub enum ImageFormat {
     Vmdk,
 }
 
+#[derive(Deserialize, Debug, Copy, Clone)]
+#[serde(try_from = "i32")]
+pub struct ImageSize(i32);
```

Comment:
Correct, it's `i32` because of `pubsys`, to avoid dealing with fallible numeric conversions. I'll see if a `u16` might work too without much boilerplate.